apis/v1alpha1/generator.yaml

```diff
@@ -3,8 +3,15 @@ ignore:
       - Activation
       - Association
       - AssociationBatch
-      - Document
+      # - Document
       - MaintenanceWindow
       - OpsItem
       - PatchBaseline
       - ResourceDataSync
+
+resources:
+  Document:
+    exceptions:
+      errors:
+        404:
+          code: InvalidDocument
```

Comment:
Does the API return 1InvalidDocument` when a document doesn't exist?

Reply:
Yes!